modules/gdscript/language_server/gdscript_workspace.h

```diff
@@ -75,9 +75,12 @@ class GDScriptWorkspace : public Reference {
 
 	String get_file_path(const String &p_uri) const;
 	String get_file_uri(const String &p_path) const;
+	String simplify_file_uri(const String &p_path) const;
```

Comment:
This method seems only called in `get_file_uri` so you can put the implementation inside `get_file_uri`